### PR TITLE
[Docs] Fix example php code in docs.

### PR DESCRIPTION
**Pull request by sfadschm**
This is the follow up of #5689.

It aims to make all php examples executable and apply our CS rules.

**Status:**
- [x] Convert non-php code.
- [x] Display opening tags (for PHP) and closing tags (for mixed HTML).
- [x] Remove occurences of . :.
- [x] Replace utf-8 characters (except the `euro` and `pound` signs).
- [ ] Fix missing class definitions.
- [ ] Fix remaining syntax errors.
- [ ] Enable and run php-cs-fixer for user_guide_src\source.
- [ ] Resolve merge conflicts.

### CODE REVIEW

File: user_guide_src/source/incoming/incomingrequest.rst [hunk 1]
```diff
@@ -67,13 +62,11 @@ will return null if the item doesn't exist, and you can have the data filtered.
 use data without having to test whether an item exists first. In other words, normally you might do something
 like this:
 
-.. literalinclude:: incomingrequest/7.php
-   :lines: 2-
+.. literalinclude:: incomingrequest/008.php
 
-With CodeIgniter’s built in methods you can simply do this:
+With CodeIgniter�s built in methods you can simply do this:
```
Comment (kenjis) [inline user_guide_src/source/incoming/incomingrequest.rst:67]: `CodeIgniter�s` ?

**Comment by kenjis** [comment]
@sfadschm 
This PR is going to be very big.
Can you break it up into smaller PRs?

File: user_guide_src/source/incoming/incomingrequest.rst (hunk 1 above)
Comment (sfadschm) [reply user_guide_src/source/incoming/incomingrequest.rst]: Fixed with the utf-8 conversion.

**Comment by sfadschm** [comment]
> @sfadschm This PR is going to be very big. Can you break it up into smaller PRs?

Yes, will do 😄

**Comment by sfadschm** [comment]
I am splitting this up and will rebase the later ones when the first ones are merged.

**Pull request closed without merging**